backends/lean/Aeneas/Progress/ProgressStar.lean

```diff
@@ -0,0 +1,326 @@
+import Lean
+import Aeneas.Utils
+import Aeneas.Progress.Progress
+import Aeneas.Progress.Core
+import Aesop.Util.Basic
+/- import Mathlib.Control.Sequence -/
+open Aeneas
+open Lean Meta Elab Tactic 
+
+namespace Bifurcation/- {{{ -/
+/-- Expression on which a branch depends -/
+structure Discr where
+  toExpr: Expr
+  /-- Name under which to bind the discriminant equation, if provided -/
+  name?: Option Name := none
+  deriving Repr
+instance: ToMessageData Discr where
```

Comment:
I like having spaces between the various definitions, even though they are related